### ISSUE TITLE
Support tag archiving

### ISSUE THREAD

**Issue by retog** [tridoc/tridoc-backend#25]
I would like to be able to hide tags from the default list of tags.

**Comment by nleanba** [comment]
Please elaborate: what do you mean with default list of tags: the response to GET /tags? Should the tags stay attached to the documents?

**Comment by retog** [comment]
Yes, the archived tags should stay attached to the documents. The idea is that clients can hide them by default but it still possible to find all documents with the tag. The use case is when a topic ceases to part of your current focus, such as when a project ends, you no longer want to see the respective tag by default but you might still want to search for it on occasion, or you might want to start working on that topic again and unarchive the tag for that purpose.
